docker/mqtt_metrics_collector/mqtt_metrics_collector.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+import os
+
+from prometheus_client import start_http_server, Counter
+
+# de-register default-collectors
+from prometheus_client import REGISTRY, PROCESS_COLLECTOR, PLATFORM_COLLECTOR
+REGISTRY.unregister(PROCESS_COLLECTOR)
+REGISTRY.unregister(PLATFORM_COLLECTOR)
+
+# remove python-gargage-collectior metrics
+REGISTRY.unregister(REGISTRY._names_to_collectors['python_gc_objects_uncollectable_total'])
+#REGISTRY.unregister(REGISTRY._names_to_collectors['python_gc_objects_collected_total'])
+
+import logging
+
+from base64 import b64decode
+import json
+import paho.mqtt.client as mqtt
+import random
+import urllib
+import urllib.request
+
+import sys
+
+WIS2BOX_LOGGING_LOGLEVEL = os.environ.get('WIS2BOX_LOGGING_LOGLEVEL')
+# gotta log to stdout so docker logs sees the python-logs
+logging.basicConfig(stream=sys.stdout)
+# create our own logger using same log-level as wis2box
+logger = logging.getLogger('mqtt_metrics_collector')
+logger.setLevel(WIS2BOX_LOGGING_LOGLEVEL)
+
+INTERRUPT = False
+
+def sub_connect(client, userdata, flags, rc, properties=None):
```

Comment:
Add function annotation and docstring (example: https://github.com/wmo-im/wis2box/blob/main/wis2box/data/__init__.py#L67-L74)